### PR TITLE
[ACTIONS] Crimeometer Added new actions

### PR DESCRIPTION
**Pull request by jcortes**
### Summary
🆕🛠️📦

This pull request adds three new action components for the Crimeometer app that retrieve raw data, raw data coverage, and statistics from the Crimeometer API. It also updates the `pnpm-lock.yaml` file and the `package.json` file for the app, and defines the app type, props, methods, and constants in separate files. It also deletes some unused files from the previous version of the app.

> _The Crimeometer app is quite neat_
> _It fetches some data on crime and heat_
> _With actions and props_
> _And `axios` for hops_
> _It's a Pipedream component complete_

### CODE REVIEW

File: components/crimeometer/actions/retrieve-raw-data-coverage/retrieve-raw-data-coverage.mjs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import constants from "../../common/constants.mjs";
+import app from "../../crimeometer.app.mjs";
+
+export default {
+  key: "crimeometer-retrieve-raw-data-coverage",
+  name: "Retrieve Raw Data Coverage",
+  description: "Provides raw data coverage information grouped by cities. [See the docs](https://www.crimeometer.com/crime-data-api-documentation) and [here](https://documenter.getpostman.com/view/12755833/TzK2auPn#ea645970-96d9-4fbd-8c29-d070f08b1f12).",
+  type: "action",
+  version: "0.0.3",
```
Comment (andrewjschuang) [inline components/crimeometer/actions/retrieve-raw-data-coverage/retrieve-raw-data-coverage.mjs:9]:
```suggestion
  version: "0.0.1",
```

**Review by andrewjschuang — CHANGES_REQUESTED** [review]
Thanks, I added a couple comments!

**Review by andrewjschuang — APPROVED** [review]
Thanks, LGTM!

**Comment by vunguyenhung** [comment]
Update: Trying the API Key provided by CrimeoMeter resulted in error
image

I'm contacting CrimeoMeter to ask for support

**Pull request merged**